bom-latest/pom.xml

```diff
@@ -141,6 +157,16 @@
                 <artifactId>display-url-api</artifactId>
                 <version>2.3.2</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>docker-commons</artifactId>
+                <version>1.5</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>docker-workflow</artifactId>
```

Comment:
I would like to do JENKINS-58732 BTW.

Reply:
https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/373 specifically

Reply:
That's ok - and indeed would let us knock a few transitive dependencies off what is quite a big change here. We can revisit this when that PR is merged

Reply:
Also the split would reduce the number of tests run in `pipeline-model-definition`, which is currently gigantic, even slower than `workflow-cps`, thus becoming the bottleneck for the `bom` parallel run.

Reply:
I noticed that jenkinsci/pipeline-model-definition-plugin#373 was merged earlier today - let me know when the next version is released, and I'll re-work this PR.

Reply:
Actually it is going to take longer than that, for purposes of this PR. See rollout plan in https://github.com/jenkinsci/docker-workflow-plugin/pull/202. I do not think you need to block this PR on it.

Reply:
I was hoping to avoid wrangling the icon-shim-plugin's inconsistent tags, which I believe docker-workflow was pulling in, by waiting. Is that unblocked at this point, or is is it still present?

Reply:
Afraid I know nothing about this `icon-shim` issue.